src/wbase/UserQueryInfo.h

```diff
@@ -89,14 +89,13 @@ class UserQueryInfo {
 
     QueryId getQueryId() const { return _qId; }
 
-    CzarIdType getCzarId() const { return _czarId; }
+    CzarId getCzarId() const { return _czarId; }
 
 private:
-    UserQueryInfo(QueryId qId, CzarIdType czId);
+    UserQueryInfo(QueryId qId, CzarId czId);
 
-    util::InstanceCount const _icUqi{"UserQueryInfo"};
     QueryId const _qId;  ///< The User Query Id number.
```

Comment:
Inconsistent naming convention for `qId`. Should be:
```
QueryId const _queryId;
```
The second observation - the comment is useless, as we know that type `QueryId` is always "The User Query Id number". We don't need to repeat this over and over again in the code. The type name is informative enough.